src/components/gridlines.ts

```diff
@@ -42,6 +64,36 @@ export class Gridlines extends Component {
     }
   }
 
+  public betweenX(_betweenX: boolean): this;
```

Comment:
It can make sense for linear as well! For example, it enables inset tick labels like this

![image](https://user-images.githubusercontent.com/2644420/27586958-6591a31e-5b3a-11e7-9496-1bbe28dcdf91.png)